packages/web/components/input/search-box.tsx

```diff
@@ -40,6 +48,11 @@ type SearchBoxProps = Omit<InputProps<string>, "currentValue"> &
   VariantProps<typeof searchBoxClasses> & {
     type?: string;
     currentValue?: string;
+    /**
+     * adds the ability to output the searchbox event using a debounce effect,
+     * which is useful to avoid making too many requests.
+     */
+    debounce?: number;
```

Comment:
So you've built debounce feature directly into the component itself? Perhaps this should be used over all `useSearchQueryInput` uses. I'll keep this in mind going forward and with new assets table

Reply:
Yes, because it allows us to avoid having a dedicated hook and allows us not to have a double state